src/assets/css/modal.css

```diff
@@ -132,6 +132,11 @@ div.modal_popup table:not(#settingsEditor) td:first-child {
     text-align: center;
 }
 
+div#settingsEditorContainer {
```

Comment:
Sorry, didn't realize that you already fixed this. You're going to have to figure out with the merger with upstream...

Reply:
Sorry, are you saying there is a conflict? I don't see it.

Reply:
I don't see the conflicts.